### ISSUE TITLE
Set text color when a button is hovered

### ISSUE THREAD

**Issue by over-engineer** [pressidium/pressidium-cookie-consent#44]
Initially suggested by @formaciongrafica in #43.

>Using the plugin I have seen that it does not have options for the hover color of the primary and secondary button texts. In some cases this can generate accessibility problems due to the low contrast between the background colors and texts of the hover states of these buttons.

>The contrast problem is in order to comply with the GDPR, offering designs on both buttons so that they do not induce the user to choose one option over another.

We should add options to set a different color for the text when the primary/secondary buttons are hovered.

**Comment by over-engineer** [comment]
This issue was addressed in `1.3.0`, which was just released on the Plugin Directory.

Thanks @formaciongrafica for your suggestion! :tada:
